webpack/scenes/ContentViews/Details/Filters/ContentViewFilters.js

```diff
@@ -46,7 +50,7 @@ const ContentViewFilters = ({ cvId }) => {
       const {
         id, name, type, description, updated_at: updatedAt, inclusion,
       } = filter;
-      if (filter.type === 'erratum' && filter.rules[0].types) errataByDate = true;
+      if (filter.type === 'erratum' && filter.rules.length && filter.rules[0].types) errataByDate = true;
```

Comment:
This line is because while working on this I noticed a failure 
![Screenshot from 2021-07-16 09-51-22](https://user-images.githubusercontent.com/21146741/125958727-1d29acb1-18ae-4435-b9ea-059ccd952fb8.png)

Reply:
Yeah, in general we need more undefined checks on our object refs. 
But question, you have the variables right above this why not use destructuring? 
      const {
        id, name, type, description, updated_at: updatedAt, inclusion,
      } = filter;
     Then you can add
      if (  (type && rules.length > 0)  && filter.type === 'erratum' &&  filter.rules[0].types) errataByDate = true;

Reply:
Updated and added tests for add..